recipe-server/normandy/recipes/tests/api/v1/test_serializers.py

```diff
@@ -220,3 +225,22 @@ def test_it_works_with_no_signature(self, rf):
                 'is_approved': False,
             }
         }
+
+
+@pytest.mark.django_db()
+class TestSignatureSerializer:
+    def test_it_works(self):
+        signature = SignatureFactory()
+        serializer = SignatureSerializer(instance=signature)
+
+        assert serializer.data == {
+            'signature': Whatever.regex(r'[a-f0-9]{40}'),
+            'x5u': Whatever.startswith(signature.x5u),
+            'timestamp': Whatever.iso8601(),
+            'public_key': Whatever.regex(r'[a-zA-Z0-9/+]{160}')
+        }
+
+    def test_it_cachebusts_x5u(self):
+        signature = SignatureFactory()
+        serializer = SignatureSerializer(instance=signature)
+        assert 'cachebust=' in serializer.data['x5u']
```

Comment:
Perhaps assert that it's in the query string rather than simply in the URL at all?